File: Timestables.c
#include <stdio.h>
int times(int chicken);

int main(){
int n;
for (n = 1; n <= 12; n++) {
printf("The times table for %d is", n);
printf("%d\n", times(n));}
}

int times(int chicken){
int i, table=1;
for (i = 1; i <= 12; i++) {
table = chicken * i;
printf("%d times %d is %d\n", chicken, i, table); }
return 0;
}

File: Timestables.java
public class Timestables{

  public static void main(String[] args) {
    System.out.println("Hello World");
    int n = 2;
    while (n <= 12) {
    printTimestables(n);
    n = n + 1; }
  }

 public static void printTimestables(int chicken) {
 System.out.println("This is the" + chicken + " times table:");
 int m = 1;
 while (m <= 12) {
 int a = chicken * m;
 System.out.println(chicken + " times " + m + " is " + a);
 m=m+1;
  }
  System.out.println();
}

}
